src/Microsoft.AspNet.JsonPatch/Adapters/ObjectAdapter.cs

```diff
@@ -107,15 +108,13 @@ public void Add(Operation<TModel> operation, TModel objectToApplyTo)
         /// Add is used by various operations (eg: add, copy, ...), yet through different operations;
         /// This method allows code reuse yet reporting the correct operation on error
         /// </summary>
-        private void Add(string path, object value, TModel objectToApplyTo, Operation<TModel> operationToReport)
+        private void Add(string path, object value, object objectToApplyTo, Operation operationToReport)
```

Comment:
why is `objectToApplyTo` `dynamic` in the above `Add` method and `object` in this one?